rpcd/playbooks/stage-python-artifacts.yml

```diff
@@ -56,6 +56,18 @@
       tags:
         - always
 
+    # TODO(odyssey4me)
+    # Remove this and adjust the fact setting once
+    # https://review.openstack.org/439659 is available
+    - name: Fetch the list of files in the releases folder
+      uri:
+        url: "{{ rpco_mirror_base_url }}/os-releases/{{ rpc_release }}/"
+        return_content: yes
+        validate_certs: "{{ https_validate_certs | bool }}"
+      register: releases_index
+      tags:
+        - always
+
     - name: Derive the lists of artifacts to download
       set_fact:
         python_wheel_list: |
```

Comment:
Please comment what a python_wheel_list, git_folder_list, python_venv_list, and releases_file_list look like